lima, limactl: add page (#10648)
tldr-pages/tldr: pages/linux/lima.md

# lima

> > This command is a wrapper for `limactl`.
> > More information: <https://github.com/lima-vm/lima>.
> 
> - View documentation of `limactl`:
> 
> `tldr limactl`

Reviewer: Since this is an alias page, we can use our [generic template](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/alias-pages.md#en) to facilitate easy translation.

```suggestion
> This command is an alias of `limactl`.
> More information: <https://github.com/lima-vm/lima>.

- View documentation for the original command:

`tldr limactl`
```

> > This command is an alias of `limactl`.
> > More information: <https://github.com/lima-vm/lima>.
> 
> - View documentation for the original command:
> 
> `tldr limactl`

Reviewer 2: https://github.com/lima-vm/lima
> #### `limactl shell`
> `limactl shell <INSTANCE> <COMMAND>`: launch `<COMMAND>` on Linux.
>
> For the "default" instance, this command can be shortened as `lima <COMMAND>`.
> The `lima` command also accepts the instance name as the environment variable `$LIMA_INSTANCE`.

```suggestion
> This command is an alias of `limactl shell` for the default VM instance.
> You can also set the `$LIMA_INSTANCE` environment variable to work on a different instance.
> More information: <https://github.com/lima-vm/lima>.

- View documentation for the original command:

`tldr limactl`
```
